config/roles.yml

```diff
@@ -355,3 +355,22 @@ security_analytics_ack_alerts:
   reserved: true
   cluster_permissions:
     - 'cluster:admin/opensearch/securityanalytics/alerts/*'
+
+# Roles for Hello World extension, roles that extensions would like to add should be ultimately be defined elsewhere
```

Comment:
Same for these. If there are specific demo configurations that we would like to ship by default would it make sense to have separate files in different folders under `config/` here? Maybe put these into `config/extension_demo/roles.yml`?